src/components/TimelinePost.vue

```diff
@@ -102,20 +97,39 @@ export default {
 			return Date.parse(this.item.published)
 		},
 		formatedMessage() {
-			var message = this.item.content
+			let message = this.item.content
+
 			if (typeof message === 'undefined') {
 				return ''
 			}
+
 			message = message.replace(/(?:\r\n|\r|\n)/g, '<br />')
-			message = message.linkify({
-				formatHref: {
-					mention: function(href) {
-						return OC.generateUrl('/apps/social/@' + href.substring(1))
-					}
-				}
-			})
-			message = this.mangleHashtags(message)
+
+                        // Create links for mentions
+			if (typeof this.item.source !== 'undefined') {
+	                        let source = JSON.parse(this.item.source)
+        	                if (typeof source.tag !== 'undefined') {
+                	                source.tag.forEach(tag => {
+                        	                if (tag.type === 'Mention') {
+							let shortname = tag.name.match(/^@[^@]+/)
+                                        	        let patt = new RegExp(shortname[0], "g")
+                                                	message = message.replace(patt, function(matched) {
+	                                                        var a = '<a href="' + tag.href + '">' + matched + '</a>'
```

Comment:
maybe we don't want to use tag.href here but rather construct a local one using OC.generateUrl?

Reply:
I have updated this in https://github.com/nextcloud/social/pull/667/commits/b5d729bda8dd3b12722d86aae5134fabc5ce9246

Reply:
i guess both works !